docs/extra-apps.md

```diff
@@ -0,0 +1,8 @@
+# Extra apps
+
+## omxplayer
+
+omxplayer depends on libav which has a commercial license. So in order to be
+able to compile omxplayer you will need to whiteflag the commercial
+licenseadding to you local.conf:
```

Comment:
This last line is a bit strange.

Maybe it should say: "So in order to be able to compile omxplayer you will need to whiteflag the commercial license in your local.conf:"